sources/osgUtil/NodeGizmo.js

```diff
@@ -599,7 +599,13 @@ NodeGizmo.prototype = MACROUTILS.objectInherit( MatrixTransform.prototype, {
     getCanvasPositionFromWorldPoint: function ( worldPoint, out ) {
         var cam = this._viewer.getCamera();
         var mat = Matrix.create();
-        Matrix.preMult( mat, cam.getViewport() ? cam.getViewport().computeWindowMatrix() : Matrix.create() );
+
+        if ( cam.getViewport() ) {
```

Comment:
Would have been a good occasion for closuring all thos temporary variables...
Perhaps that function could be in View or Viewer
(duplicate here https://github.com/sketchfab/showwebgl/commit/cde9ca749051e3ea9bb7346ba5444554b5c335b5 )